README.md

```diff
@@ -26,3 +33,8 @@ I will be using issues to keep track of bugs and features, both created by the c
 
 v0.01 - Init commit, very strict usage
 
+## Todo:
+Update mod json file automatically from other repository using same bot
+Fix some stuff where some Primed mods are still detected as their normal counterparts
+Add Riven OCR detection
```

Comment:
these should probably be issues instead of a list in the readme

Reply:
Correct, but I didn't want to add issues here if my pull request wasn't merged yet, and you cant make issues on a fork

Reply:
Nevermind you can, you just have to enable it